lib/algos/list/basic.js

```diff
@@ -1,12 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+const Extension = require('./Extension').default;
+
 const checkLimit = require('./tools').checkLimit;
 const DEFAULT_MAX_KEYS = 10000;
 
 /**
  *  Class of an extension doing the simple listing
  */
-class List {
+class List extends Extension {
```

Comment:
`extends` currently performs poor in pure ES6 implementation. You can choose to use the class directly instead or introduce base class in a PR later when performance regression is fixed in node 6.